orchagent/port.h

```diff
@@ -11,6 +11,7 @@ extern "C" {
 #include <map>
 
 #define DEFAULT_PORT_VLAN_ID    1
+#define DEFAULT_IF_MTU        9100
```

Comment:
why do we have DEFAULT_IF_MTU in two places? can we have them in one place?

Reply:
sure. will combine these two definitions in one header file. Thanks